sections/slideshow.liquid

```diff
@@ -0,0 +1,465 @@
+            <img
+              srcset="{%- if block.settings.image.width >= 375 -%}{{ block.settings.image | img_url: '375x' }} 375w,{%- endif -%}
+              {%- if block.settings.image.width >= 750 -%}{{ block.settings.image | img_url: '750x' }} 750w,{%- endif -%}
+              {%- if block.settings.image.width >= 1100 -%}{{ block.settings.image | img_url: '1100x' }} 1100w,{%- endif -%}
+              {%- if block.settings.image.width >= 1500 -%}{{ block.settings.image | img_url: '1500x' }} 1500w,{%- endif -%}
+              {%- if block.settings.image.width >= 1780 -%}{{ block.settings.image | img_url: '1780x' }} 1780w,{%- endif -%}
+              {%- if block.settings.image.width >= 2000 -%}{{ block.settings.image | img_url: '2000x' }} 2000w,{%- endif -%}
+              {%- if block.settings.image.width >= 3000 -%}{{ block.settings.image | img_url: '3000x' }} 3000w,{%- endif -%}
+              {%- if block.settings.image.width >= 3840 -%}{{ block.settings.image | img_url: '3840x' }} 3840w,{%- endif -%}
+              {{ block.settings.image | img_url: 'master' }} {{ block.settings.image.width }}w"
+              sizes="100vw"
+              src="{{ block.settings.image | img_url: '1500x' }}"
+              loading="lazy"
+              alt="{{ block.settings.image.alt | escape }}"
+              width="{{ block.settings.image.width }}"
+              height="{{ block.settings.image.width | divided_by: block.settings.image.aspect_ratio | round }}"
+            >
+          {%- else -%}
+            {{ 'lifestyle-2' | placeholder_svg_tag: 'placeholder-svg' }}
+          {%- endif -%}
+        </div>
+        <div class="slideshow__text-wrapper slideshow__text-wrapper--{{ section.settings.box_align }} banner__content page-width">
+          <div class="slideshow__text banner__box color-{{ section.settings.color_scheme }}">
+            {%- if block.settings.heading != blank -%}
+              <h2 class="banner__heading {% if section.settings.heading_size == 'medium' %}h1{% else %}h0{% endif %}">{{ block.settings.heading | escape }}</h2>
+            {%- endif -%}
+            {%- if block.settings.subheading != blank -%}
+              <div class="banner__text" {{ block.shopify_attributes }}>
+                <span>{{ block.settings.subheading | escape }}</span>
+              </div>
+            {%- endif -%}
+            {%- if block.settings.button_label != blank -%}
+              <a{% if block.settings.link %} href="{{ block.settings.link }}"{% else %} role="link" aria-disabled="true"{% endif %} class="button {% if block.settings.button_style_secondary %}button--secondary{% else %}button--primary{% endif %}">{{ block.settings.button_label | escape }}</a>
+            {%- endif -%}
+          </div>
+        </div>
+      </div>
+    {%- endfor -%}
+  </div>
+
+  {%- if section.blocks.size > 1 -%}
+    <div class="slideshow__controls slider-buttons no-js-hidden color-{{ section.settings.color_scheme }}">
```

Comment:
@svinkle, the fact that the slideshow controls is below the images makes sense on desktop I believe but is it an issue for mobile ? 

Since some of the content of the slide, depending on the section settings, can be below the control bar you will tab to the slide button that is below the controls first and then get to the controls. 
[video](https://screenshot.click/21-11-tw714-6rtw7.mp4) (I'm fixing the focus ring now)

Reply:
Right I noticed this, too. Are you able to adjust the placement of the controls in the DOM to match the visual order?

Reply:
So I'd see two options and curious to hear your thoughts:  

- 1) move the controls above the image and text box container (slide container). Which would work as expected on mobile. But on desktop it'd mean that focus would first go on the controls (below the slide) and then back up onto the button of the slide, if any, and then onto whatever is the next section/focusable element. [Video of tab order](https://screenshot.click/21-11-4vd5y-znn6n.mp4)
- 2) duplicate the controls code and have one for mobile that is before the slides and then one on desktop after the slides 🤔

Reply:
Curious to know, which is worse in terms of perf; duplicate controls or moving the set of controls in the DOM when a breakpoint condition is met?

@melissaperreault, @danielvan curious to understand the rationale behind the slideshow placement on mobile. It's resulting in some friction on the dev side of things. Could we normalize and keep in the same position?

Reply:
Like on Debut, the controls are placed in this position to maintain its position regardless of the slide content length. Otherwise we lose the controls as you navigate the slides, moving up or down, sometimes in the viewport, sometimes outside. I can share a quick prototype we've made to base this decision from.

Reply:
Yeah I can see how it's useful on mobile specifically when having the text box below the image. Cause then the content can be quite long vertically. 

In terms of the HTML structure... I think it makes sense to keep the controls after the slideshow because that's where it needs to be on desktop and also on mobile when you set the text box to be over the image and not below it.  So I only need to tweak the positioning of the controls for one situation. 

Could I change the tabbing order specifically in that one situation @svinkle ? I feel like that's something we would ideally avoid but it might make sense here 🤔